e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -59,13 +62,13 @@ public void When_ExtractRequestReceived_Expect_ExtractStatusAndDocumentDataAdded
         String ehrExtractRequest = buildEhrExtractRequest(conversationId, EXISTING_PATIENT_NHS_NUMBER, FROM_ODS_CODE_1);
         MessageQueue.sendToMhsInboundQueue(ehrExtractRequest);
 
-        assertHappyPathWithDocs(conversationId, FROM_ODS_CODE_1);
+        assertHappyPathWithDocs(conversationId, FROM_ODS_CODE_1, EXISTING_PATIENT_NHS_NUMBER, DOCUMENT_ID_NORMAL);
```

Comment:
I'd rename EXISTING_PATIENT_NHS_NUMBER to something more consistent to avoid confusion